agent/ssi/agent.go

```diff
@@ -23,6 +26,7 @@ type AgentType interface {
 type Agent interface {
 	AgentType
 	Wallet() (h int)
+	ManagedWallet() managed.Wallet
```

Comment:
.. as seen here as well, we have two different `Wallet()` and `ManagegWallet()`. For first-timer, this might be difficult == future topic.

Reply:
Yes, I will add some comments that this is only temporary solution.